### PR TITLE
First simple mock-up of new homepage [L] #335

### PR DESCRIPTION
**Pull request by nathenf**
Includes copy and content structure for new homepage with content ranked using prioritized job stories

### CODE REVIEW

**Comment by rufuspollock** [comment]
Re PR summary (which is super useful): also helpful you had to include tao stuff re community and why (e.g. did not know how to have 2 PRs at same time).

File: vault/notes/new-home-page.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+Slider including blog posts and podcasts
+
+Sign Up to our newsletter and don't miss out on any new updates
+[[sign up form]]
+
+![[../assets/images/profile-image-zaib.jpg]]
+
+## Meet The Team
+
+Our team consists of artists, researchers,
```
Comment (rufuspollock) [inline vault/notes/new-home-page.md:95]: 0️⃣ Again, any reason for line-wrapping (0️⃣ means low priority comment).

File: vault/notes/new-home-page.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+Inspired by integral theory, zen buddhism and other key concepts, we cultivate these ideas and 
+apply them to life itself, through deliberately developmental programs and community building.
+
+### [[../about |Learn More]]
+
+____________
+
+## Join The Community
+
+The Life Itself online community space is a fantastic place to get started.
```
[the remark this answers is not in the document]
Comment (rufuspollock) [reply vault/notes/new-home-page.md:37]: Maybe also some pictures / profiles of people in the community.

File: vault/tao/community-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+The Life Itself community chat is a space for members of the Life Itself community to gather online. [Join the community chat here](https://chat.whatsapp.com/JNJCTZugNQn1fq89xbHtfA) or by scanning the QR Code below. 
+
+![[../assets/images/Whatsapp General Chat QR Code.png |400]]
+
+## Purpose
+
+The Life Itself Community Chat is an online space for members of the Life Itself community for:
+
+1.  👥 Connection - Through the community chat we hope to connect deeply together and develop new relationships. Most importantly we encourage openness for a diversity of perspectives and connecting with people who think and do differently and yet aspire to the same need for change.
```
Comment (rufuspollock) [inline vault/tao/community-guidelines.md:16]: 1️⃣ formatting: 2 spaces are bullet item.

File: vault/notes/new-home-page.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+commitment to being wise and taking practical
+action to solve the collective challenges facing
+our society and culture. Read more about our 
+people here.
+
+### [[../people |Read More]]
+
+## Our Journey So Far
+
+Timeline of our journey including links to resources along the way.
```
Comment (rufuspollock) [inline vault/notes/new-home-page.md:107]: Can we have a go at actually drafting a basic version of this? I think we have material from 2020 gathering we could use and could do a quick interview.

Also is this about a **history** (timeline) or a **story** (as in our motivation ...)

File: vault/tao/community-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
+---
+title: "Life Itself Community Chat Guidelines"
```
Comment (rufuspollock) [inline vault/tao/community-guidelines.md:2]: Are these just chat guidelines or our community guidelines?

File: vault/tao/community-guidelines.md
Comment (rufuspollock) [inline vault/tao/community-guidelines.md:1]: Overall these are excellent 👏

**Review by rufuspollock — APPROVED** [review]
Great start. Because i think making changes will be tough, i think i just merge and then we iterate (e.g. you can just rename the image file).

File: vault/notes/new-home-page.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
+---
+title: "New Home Page"
+created: 2023-03-28
+---
+
+## Ever asked yourself ... How can I live a happier life? Create a better society?
+
+We’re an open community dedicated to the art of wiser living and social transformation. We favor approaches that prioritize inner development and cultural change in a rigorous, practical way.  
+
+Sign up to our newsletter for latest updates, projects
```
[the remark this answers is not in the document]
Comment (nathenf) [reply vault/notes/new-home-page.md:10]: By line wrapped do you mean []? I was looking for a way to highlight sections which would be replaced with a piece off code. Perhaps I can just bold it instead.

File: vault/tao/community-guidelines.md (hunk 3 above)
Comment (nathenf) [reply vault/tao/community-guidelines.md:16]: This is not clear for me. Do you mean I should add 2 spaces for the numbered list? or something else?

**Pull request merged**